src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -60,6 +61,10 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkVersion)" />
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="$(SystemDiagnosticsDiagnosticSourceVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
Not sure this should be needed if it wasn't there already.

Reply:
I could not run any tests within visual studio without it.

Reply:
But I can remove it just before I squash the final push if you think?

Reply:
Please remove this PackageReference before the final merge. We have 
`<PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />`
in src\Microsoft.Data.SqlClient\tests\tools\Microsoft.DotNet.XUnitExtensions\Microsoft.DotNet.XUnitExtensions.csproj which will be picked up by msbuild.

Reply:
ok, it just makes it impossible to run any tests from within visual studio, which is a shame.
But will do.

Reply:
Odd. I thought the tests ran inside visual studio before. If not then can you file an issue for it? I run the tests from the commandline because that's how I'm used to working with the project from back when it was part of corefx.